src/ml/neural_net/mps_layer_helper.h

```diff
@@ -15,7 +15,7 @@ NS_ASSUME_NONNULL_BEGIN
 @class TCMPSInstanceNormDataLoader;
 
 @interface MPSCNNFullyConnectedNode (TCMPSLayerHelper)
-@property (nonatomic, strong) TCMPSConvolutionWeights *weights API_AVAILABLE(macosx(10.14));
+@property (nonatomic, strong) TCMPSConvolutionWeights *tcWeightsData API_AVAILABLE(macosx(10.14));
```

Comment:
Let's just go ahead and follow the officially documented naming guidelines, which would yield something like `tc_weights` or `tc_weightsData`. See "Avoiding Category Method Name Clashes" here: https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/CustomizingExistingClasses/CustomizingExistingClasses.html